src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/roi_align.cpp

```diff
@@ -8,6 +8,32 @@ namespace {
 using ov::test::ROIAlignLayerTest;
 using ov::test::ROIAlignV9LayerTest;
 
+class ROIAlignLayerGPUTest : public ROIAlignLayerTest {
+protected:
+    void SetUp() override {
+        ROIAlignLayerTest::SetUp();
+        auto param = this->GetParam();
+        ov::element::Type model_type = std::get<7>(param);
+        if (model_type == ov::element::f32) {
```

Comment:
do we need this condition? 
if it is other precisions like f16, the threshold should be more critical.

Reply:
Yes, you're right. The condition was added as f16 and f32 are expected to require different threshold values. Do you think other approach?